### DISCUSSION TITLE
Better interfacing of Blender and MDAnalysis

### DISCUSSION THREAD

**Discussion by ShivaniDM** [MDAnalysis/mdanalysis#4512] [GSoC Discussions]
@BradyAJohnston
@yuxuanzhuang

Looking forward to hear from you on the pre-proposal for project 5:Better interfacing of Blender and MDAnalysis.
also for the interactive part of the problem will the script  run in blender  through bpy ?

**Comment by natecross123** [comment]
Hello good day what are the best steps to take for me to be a competitive canidate

**Reply by hmacdope** [reply]
See our outreachy guide on the wiki: https://github.com/MDAnalysis/mdanalysis/wiki/Outreachy

**Comment by hmacdope** [comment]
@ShivaniDM you will hear back on your proposal as soon as possible.
